src/constants/index.ts

```diff
@@ -2,3 +2,21 @@ export const TIME_30D = 30 * 24 * 60 * 60
 export const TIME_1D = 24 * 60 * 60
 export const TIME_1H = 60 * 60
 export const TIME_10S = 10
+
+export const IsMainnet = process.env.NET_ENV === 'production'
+
+export const MainNet = {
+  indexer: 'https://indexer-not-use-in-production-env.did.id/',
```

Comment:
Do not use `xxx-not-use-in-production` anymore. It will be removed in the future. Use `indexer-v1` instead.

Check here: https://github.com/dotbitHQ/dotbit.js/blob/main/src/createInstance.ts#L14